<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Diaries</class>
 <widget class="QWidget" name="Diaries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>691</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>181</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>字体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>字号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditFontSize"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>160</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QToolButton" name="toolButtonColor">
        <property name="text">
         <string>颜色</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/color.png</normaloff>:/images/res/pic/color.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonUnderline">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>下划线</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/underline.png</normaloff>:/images/res/pic/underline.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSave">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/save.png</normaloff>:/images/res/pic/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="toolButtonBold">
        <property name="text">
         <string>粗体</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/bold.png</normaloff>:/images/res/pic/bold.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonItalic">
        <property name="text">
         <string>斜体</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/italic.png</normaloff>:/images/res/pic/italic.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonPicture">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="pic.qrc">
          <normaloff>:/images/res/pic/picture.png</normaloff>:/images/res/pic/picture.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
